Revamped signup page as asked by you

harbakshsinghbaath opened the pull request:
Updated the css file of signup page

# Pull Request Format
## PR Title
### Issue #69  : **signup page revamp** solved

## Type of PR
- **Add X in the box to specify the improvement type.**
- [] Bug fix
- [ X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): ___________  


## Description
This pull request aims to improve the overall style of the signup page. It now not only goes better with the website but also the login page to maintain cohesion

## Screenshots / Videos (if applicable)
**Before:**
![image](https://github.com/user-attachments/assets/ad470927-7a7e-45c9-b1e7-519faa8507bb)

bland and boring

**After:**
<img width="721" alt="image" src="https://github.com/user-attachments/assets/317a4144-517f-4c5b-944f-8f02b9742407">

looks more pleasing to eye
matches with the login page
## Checklist
- **Add X in the box to specify.**
- [X] I have performed a self-review of my code.
- [ X] I have tested the changes thoroughly before submitting this pull request.
- [ X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ ] I have commented my code, particularly in hard-to-understand areas.



Thank you for reviewing my pull request!

swaraj-das commented:
@harbakshsinghbaath The sign-up page is not responsive. Please ensure it adapts properly to different screen sizes. Also, add an underline effect when hovering over the "Forgot Password" link on the login page. Don't forget to include a hover effect on the "Sign Up" link in the login page as well.

harbakshsinghbaath commented:
Hey, i have fixed the alignment adaptivity issues and added the underline effect in the sign up and forgot password in login page... they might be a bit different than what you expected but do look at it i think itll suit the overall design language of the page
